frontend/projects/upgrade/src/app/core/stratification-factors/store/stratification-factors.model.ts

```diff
@@ -0,0 +1,20 @@
+import { AppState } from '../../core.state';
+import { EntityState } from '@ngrx/entity';
+
+export interface State extends AppState {
+  stratificationFactors: StratificationFactorsState;
+}
+
+export interface StratificationFactor {
```

Comment:
is this the same object as the `FactorStrata` model in the backend? Can we use a consistent naming if so? That object has `value` and this has `values`, but I think it's supposed to be the same.

I have same question as with that model, is `Record<string, number>` the right type? Maybe I'm not understanding the function of this particular object. It would probably help if the word `value` was more descriptive, value could mean anything.

Reply:
resolved

Reply:
we are using `Record<string, number>` coz we don't have specific key names in factor like for `favourite_food` it can have pizza, burger, pie etc... while for graduated it can have yes, no with count for that value. other issue you mentioned here are resolved.

Reply:
ok i see, this number here literally for the _counts_, ok